packages/muelu/src/Graph/BrickAggregation/MueLu_BrickAggregationFactory_decl.hpp

```diff
@@ -63,6 +63,7 @@
 #include "MueLu_Aggregates_fwd.hpp"
 #include "MueLu_Exceptions.hpp"
 #include "MueLu_Utilities_fwd.hpp"
+#include "MueLu_GraphBase.hpp"
```

Comment:
Should the include be a forward file instead?

Reply:
That's not the pattern in Uncoupled, but we don't actually need it.  Fixed.